### PR TITLE
Remove option to disable precompiled shaders

### PR DESCRIPTION
**Pull request by hevrard**
This option was a leftover from GLES.

Bug: b/155288941

### CODE REVIEW

**Comment by hevrard** [comment]
> Also remove the the flag from service.proto.

Done.

File: cmd/gapit/trace.go [hunk 1]
```diff
@@ -237,7 +236,6 @@ func (verb *traceVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		ObserveDrawFrequency:         uint32(verb.Observe.Draws),
 		StartFrame:                   uint32(verb.Start.At.Frame),
 		FramesToCapture:              uint32(verb.Capture.Frames),
-		DisablePcs:                   verb.Disable.PCS,
 		RecordErrorState:             verb.Record.Errors,
```
Comment (chrisforbes) [inline cmd/gapit/trace.go:239]: This is dead now too, right?

Comment (hevrard) [reply cmd/gapit/trace.go:239]: Good catch, created b/155718960 to track this and all GL error related leftovers.

File: gapis/service/service.proto [hunk 2]
```diff
@@ -1119,8 +1119,7 @@ message TraceOptions {
   uint32 start_frame = 12;
   // How many frames should we capture
   uint32 frames_to_capture = 13;
-  // Disable disable pre-compiled shaders.
-  bool disable_pcs = 14;
+  reserved 14;
```
Comment (pmuetschard) [inline gapis/service/service.proto:1122]: Remove it in the `DeviceTraceConfiguration` message as well.

Comment (hevrard) [reply gapis/service/service.proto:1122]: Woops, sorry for missing this one out, done.

**Pull request merged**